ui-core/src/styles/select.css

```
@@ -6,6 +6,10 @@
     0px 2px 4px rgba(0, 0, 0, 0.1);
   height: 2.5rem;
   padding-left: 0.813rem;
+  padding-right: 1.875rem;
```

We could win one line of code

```suggestion
  padding-inline: 0.813rem 1.875rem;
```